addons/point_of_sale/static/src/app/models/pos_order.js

```diff
@@ -944,7 +944,7 @@ export class PosOrder extends Base {
     }
     //FIXME remove this
     getShippingDate() {
-        return this.shipping_date;
+        return this.shipping_date?.toFormat?.(localization.dateFormat) || "";
```

Comment:
Can you try using `formatDate` from web ?
```suggestion
        return this.shipping_date ? formatDate(shipping_date) : "";
```

Reply:
Hello @adgu-odoo can you please chexk again ?